client/my-sites/domains/components/domain-warnings/index.jsx

```diff
@@ -285,7 +323,9 @@ export class DomainWarnings extends PureComponent {
 				key={ expiredDomainsCanManageWarning }
 				text={ text }
 			>
-				{ this.renewLink( expiredDomains, this.onExpiredDomainsNoticeClick ) }
+				{ expiredDomainsNotAuctionLocked.length
```

Comment:
```suggestion
				{ expiredDomainsNotAuctionLocked.length > 0
```
Nit suggestion: I think this is easier to process at a glance 🙂